package.json

```diff
@@ -1,57 +1,63 @@
 {
-  "name": "t3",
-  "version": "0.1.0",
-  "private": true,
-  "scripts": {
-    "build": "next build",
-    "dev": "next dev",
-    "postinstall": "prisma generate",
-    "lint": "next lint",
-    "start": "next start"
-  },
-  "dependencies": {
-    "@headlessui/react": "^1.7.15",
-    "@heroicons/react": "^2.0.18",
-    "@next-auth/prisma-adapter": "^1.0.5",
-    "@prisma/client": "^4.14.0",
-    "@t3-oss/env-nextjs": "^0.3.1",
-    "@tanstack/react-query": "^4.29.7",
-    "@trpc/client": "^10.26.0",
-    "@trpc/next": "^10.26.0",
-    "@trpc/react-query": "^10.26.0",
-    "@trpc/server": "^10.26.0",
-    "jotai": "^2.1.1",
-    "next": "^13.4.2",
-    "next-auth": "^4.22.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-icons": "^4.9.0",
-    "react-select": "^5.7.3",
-    "superjson": "1.12.2",
-    "zod": "^3.21.4"
-  },
-  "devDependencies": {
-    "@types/eslint": "^8.37.0",
-    "@types/node": "^18.16.0",
-    "@types/prettier": "^2.7.2",
-    "@types/react": "^18.2.6",
-    "@types/react-dom": "^18.2.4",
-    "@typescript-eslint/eslint-plugin": "^5.59.6",
-    "@typescript-eslint/parser": "^5.59.6",
-    "autoprefixer": "^10.4.14",
-    "eslint": "^8.40.0",
-    "eslint-config-next": "^13.4.2",
-    "postcss": "^8.4.21",
-    "prettier": "^2.8.8",
-    "prettier-plugin-tailwindcss": "^0.3.0",
-    "prisma": "^4.14.0",
-    "tailwindcss": "^3.3.0",
-    "typescript": "^5.0.4"
-  },
-  "ct3aMetadata": {
-    "initVersion": "7.13.1"
-  },
-  "prisma": {
-    "seed": "npx ts-node prisma/seed.ts"
-  }
+   "name": "t3",
+   "version": "0.1.0",
+   "private": true,
+   "scripts": {
+      "build": "next build",
+      "dev": "next dev",
+      "postinstall": "prisma generate",
+      "lint": "next lint",
+      "prepare": "husky install",
+      "start": "next start"
+   },
+   "dependencies": {
+      "@headlessui/react": "^1.7.15",
+      "@heroicons/react": "^2.0.18",
+      "@next-auth/prisma-adapter": "^1.0.5",
+      "@prisma/client": "^4.14.0",
+      "@t3-oss/env-nextjs": "^0.3.1",
+      "@tanstack/react-query": "^4.29.7",
+      "@trpc/client": "^10.26.0",
+      "@trpc/next": "^10.26.0",
+      "@trpc/react-query": "^10.26.0",
+      "@trpc/server": "^10.26.0",
+      "jotai": "^2.1.1",
+      "next": "^13.4.2",
+      "next-auth": "^4.22.1",
+      "react": "18.2.0",
+      "react-dom": "18.2.0",
+      "react-icons": "^4.9.0",
+      "react-select": "^5.7.3",
+      "superjson": "1.12.2",
+      "zod": "^3.21.4"
+   },
+   "devDependencies": {
+      "@types/eslint": "^8.37.0",
+      "@types/node": "^18.16.0",
+      "@types/prettier": "^2.7.2",
+      "@types/react": "^18.2.6",
+      "@types/react-dom": "^18.2.4",
+      "@typescript-eslint/eslint-plugin": "^5.59.6",
+      "@typescript-eslint/parser": "^5.59.6",
+      "autoprefixer": "^10.4.14",
+      "eslint": "^8.40.0",
```

Comment:
Maybe you can show me how to properly use eslint, but it was giving me an error I could not get rid of and that error was now allowing me to move forward on whatever feature I was working on. Thats why Wayne and I deleted eslint stuff, it wasn't helping right now, since idk how to use it correctly. Was getting some kind of install conflicts with other packages.

Reply:
Sure, I'm happy to pair on this.

Reply:
Maybe we'll see other in person later today?

Reply:
Maybe we'll see other in person later today?

Reply:
Also, `eslint` was already in this repository before this change. If we do decide to remove it, we should do so in a separate pull request.